### ISSUE TITLE
Agent software sorting

### ISSUE THREAD

**Issue by tremor021** [wh1te909/tacticalrmm#651]
When you click on agent and go to software tab in bottom panel, applications can only be sorted by Name and Publisher.

Can we enable sorting by Installed On, Size and Version as well?

**Comment by wh1te909** [comment]
yes need to implement a proper sort though, that's why I disabled them on these fields because otherwise it would just sort alphabetically and not by actual size or version

**Comment by NiceGuyIT** [comment]
Example screenshot showing the importance of this issue. What is the latest version? When was it installed?
![image](https://github.com/amidaware/tacticalrmm/assets/7763429/21753906-ed19-4d35-87bd-6330b6e4877d)
